pkg/pillar/zedcloud/deferred.go

```diff
@@ -24,6 +27,7 @@ import (
 // or AddDeferred to build a queue for each key
 
 type deferredItem struct {
+	added         time.Time
```

Comment:
"added" doesn't mean much to me. Time which was added??
I can tell from the code that it appears to be some form of priority time so we send older items first,
Is that the case?